smbios.adoc

```diff
@@ -29,67 +29,79 @@ on top of cite:[SMBIOS], and is optional and recommended for BRS-B. Additional n
 not implemented. The combination of Machine Architecture ID and Machine Vendor
 ID should uniquely identify the type of hart microarchitecture that is implemented.
-| 1Bh| 000Ah| Machine Implementation ID| QWORD| Varies| Unique encoding
+| 1Bh| 0100h| Machine Implementation ID| QWORD| Varies| Unique encoding
 of the version of the processor implementation. Value of 0 is possible to indicate
 the field is not implemented. The Implementation value should reflect the design of
 the RISC-V Hart.
-| 23h| 000Ah| Instruction set supported| DWORD| Bit-field | Bits [25:0]
+| 23h| 0100h| Instruction set supported| DWORD| Bit-field | Bits [25:0]
 encodes the presence of RISC-V standard extensions, which is equivalent to bits
 [25:0] in RISC-V Machine ISA Register (**misa** CSR). Bits set to one mean the
 certain extensions of instruction set are supported on this hart.
-| 27h| 000Ah| Privilege Level Supported| BYTE| Varies| The privilege levels
+| 27h| 0100h| Privilege Level Supported| BYTE| Varies| The privilege levels
 supported by this RISC-V Hart. +
 Bit 0 Machine Mode +
 BIT 1 Reserved +
 BIT 2 Supervisor Mode +
 Bit 3 User Mode +
 BIT 6:4 Reserved +
 BIT 7 Debug Mode
-| 28h| 000Ah| Machine Exception Trap Delegation Information| QWORD| Varies|
+| 28h| 0100h| Machine Exception Trap Delegation Information| QWORD| Varies|
 Bit set to one means the corresponding exception is delegated to supervisor execution
 environment. Otherwise, supervisor execution environment must register the event
 handler in Machine-Mode for the certain exceptions through environment call.
-| 30h| 000Ah| Machine Interrupt Trap Delegation Information| QWORD| Varies|
+| 30h| 0100h| Machine Interrupt Trap Delegation Information| QWORD| Varies|
 Bit set to one means the corresponding interrupt is delegated to supervisor execution
 environment. Otherwise, supervisor execution environment must register the event
 handler in Machine-Mode for the certain interrupts through environment.
-| 38h| 000Ah| The register width (XLEN)| BYTE| ENUM| The width of register
+| 38h| 0100h| The register width (XLEN)| BYTE| ENUM| The width of register
 supported by this RISC-V Hart
-| 39h| 000Ah| Machine Mode native base integer ISA width (M-XLEN)| BYTE| ENUM|
+| 39h| 0100h| Machine Mode native base integer ISA width (M-XLEN)| BYTE| ENUM|
 The width (See below) of Machine Mode native base integer ISA supported by this RISC-V
 Hart
-| 3Ah| 000Ah| Reserved| BYTE| ENUM| Placeholder for Hypervisor Mode
-| 3Bh| 000Ah| Supervisor Mode native base integer ISA width (S-XLEN)| BYTE| ENUM|
+| 3Ah| 0100h| Reserved| BYTE| ENUM| Placeholder for Hypervisor Mode
+| 3Bh| 0100h| Supervisor Mode native base integer ISA width (S-XLEN)| BYTE| ENUM|
 The width (See below) of Supervisor Mode native base integer ISA supported by this RISC-V
 Hart
-| 3Ch| 0000Ah| User Mode native base integer ISA width (U-XLEN)| BYTE| ENUM| The
+| 3Ch| 00100h| User Mode native base integer ISA width (U-XLEN)| BYTE| ENUM| The
```

Comment:
an extra leading 0

Reply:
These fields are ultimately deleted in the patch set.